lib/util.js

```diff
@@ -11,6 +11,13 @@ const semver = require('semver');
 const MS_IN_QTR = 91 * 24 * 60 * 60 * 1000;
 exports.MS_IN_QTR = MS_IN_QTR;
 
+// Default policy for evaluating semVer
+module.exports.PRIMARY_POLICY = {
```

Comment:
I'd rename this to `DEFAULT_PRIMARY_POLICY` so it's clear this is just the default and shouldn't be modified directly